lib/matplotlib/legend.py

```diff
@@ -134,14 +136,15 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
     corner of the legend in axes coordinates (in which case
     ``bbox_to_anchor`` will be ignored).
 
-bbox_to_anchor : `.BboxBase` or pair of floats
+bbox_to_anchor : :class:`matplotlib.transforms.BboxBase` instance \
```

Comment:
The original link to `.BboxBase` should have been fine as it was.  Saves you having to continue the line.